importdb/tx.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/likecoin/likecoin-chain/v2/app"
+	"github.com/likecoin/likechain/app"
```

Comment:
Why this is changed back to `likechain`?

Reply:
It has been updated a few days ago: https://github.com/likecoin/likecoin-chain/releases/tag/v3.0.0

Reply:
we did this during nft-dev where oursky has not commit a `v3` package yet
so an extra commit to update to `v3` and fixing other breaking change in `v3.0.0-rc1` would be needed now

Reply:
updated